pkg/api/apiClient.go

```diff
@@ -17,6 +18,16 @@ func CreateAuthenticatedContext(parentCtx context.Context, apiKey string, baseUr
 
 func getApiClient() *DefaultApiService {
 	cfg := NewConfiguration()
+
```

Comment:
We can add this custom client only if there is a global flag of ignore ssl, meanwhile I leave it like this